docs/js/icons.js

```diff
@@ -1,31 +1,31 @@
 var icons = [
-  'r6',
-  'daytona',
-  'zx6r',
-  'gsxr',
-  'cbr600rr'
-]
+  "r6",
+  "daytona",
+  "zx6r",
+  "gsxr",
+  "cbr600rr",
```

Comment:
lmao!! good call, this is actually OLLDDD code from a person site. I'm going to remove this.